tests/check/test_sinks.py

```
@@ -0,0 +1,131 @@
+            success=False,
+            error="Some error",
+            message="Test failed",
+            url="http://example.com/2",
+            type_test="example",
+        ),
+    ]
+    tmp_path = Path(__file__).parent / "out"
+    output_file = tmp_path / "output.yml"
+    write_yml(results, str(output_file))
+
+    with open(output_file, "r") as f:
```

Ruff

Unnecessary open mode parameters

Remove open mode parameters

(UP015)